src/components/ItemsSection.tsx

```diff
@@ -69,14 +59,42 @@ const ItemsSection = ({ title, itemsData, category }: ItemsSectionProps) => {
     setSelectedCategory(category);
   };
 
+  // useEffect(() => {
+  //   const loadItems = async () => {
+  //     try {
+  //       const auth = getAuth();
+  //       const userId = auth.currentUser?.uid;
+
+  //       console.log('Pobrany userId:', userId);
+
+  //       if (!userId) {
+  //         console.error('User not authenticated.');
+  //         return;
+  //       }
+
+  //       const fetchedItems = await getItemsFromFirebase(
+  //         selectedCategory,
+  //         userId,
+  //       );
+  //       console.log('Pobrane przedmioty w useEffect:', fetchedItems);
+
+  //       if (fetchedItems) {
+  //         setItems(fetchedItems);
+  //       }
+  //     } catch (error) {
+  //       console.error('Error loading items:', error);
+  //     }
+  //   };
+
+  //   loadItems();
+  // }, [selectedCategory]);
```

Comment:
Do repozytorium unikajmy wysyłania zakomentowanego kodu, chyba że ma konkretny powód. Najlepiej dodawać komentarze na górze po co, dlaczego. Może tutaj nie ma to wartości, ale na projekcie z innymi developerami ma ogromną wartość "po co, i dlaczego"